setup.py

```diff
@@ -78,7 +78,7 @@
     "codecov",
     "coverage>=4.4",
     "flake8>=3.8.2",
-    "imageio>=2.5.0",
+    "imageio>=2.5.0,<2.9.0",
```

Comment:
With more recent versions of imageio, the `test_image_input_extract` test fails with this error:

```
Traceback (most recent call last):
  File "/Users/myszon/Projects/BentoML/bentoml/server/model_app.py", line 403, in api_func
    resp = api.handle_request(req)
  File "/Users/myszon/Projects/BentoML/bentoml/service/inference_api.py", line 304, in handle_request
    results = self.infer((inf_task,))
  File "/Users/myszon/Projects/BentoML/bentoml/service/inference_api.py", line 241, in infer
    user_args = self.input_adapter.extract_user_func_args(inf_tasks)
  File "/Users/myszon/Projects/BentoML/bentoml/adapters/image_input.py", line 215, in extract_user_func_args
    img_array = imageio.imread(task.data, pilmode=self.pilmode)
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/__init__.py", line 97, in imread
    return imread_v2(uri, format=format, **kwargs)
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/v2.py", line 226, in imread
    with imopen(uri, "ri", **imopen_args) as file:
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/core/imopen.py", line 259, in imopen
    plugin_instance = config.plugin_class(request, **kwargs)
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/config/plugins.py", line 107, in partial_legacy_plugin
    return LegacyPlugin(request, legacy_plugin)
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/core/legacy_plugin_wrapper.py", line 80, in __init__
    if not self._format.can_read(request):
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/core/format.py", line 242, in can_read
    return self._can_read(request)
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/plugins/tifffile.py", line 394, in _can_read
    _tifffile.TiffFile(request.get_file(), **request.kwargs)
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/plugins/_tifffile.py", line 1822, in __init__
    fh = FileHandle(arg, mode="rb", name=name, offset=offset, size=size)
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/plugins/_tifffile.py", line 4958, in __init__
    self.open()
  File "/Users/myszon/.pyenv/versions/3.8.8/lib/python3.8/site-packages/imageio/plugins/_tifffile.py", line 5022, in open
    self._fh.seek(self._offset, 2)
TypeError: seek() takes 2 positional arguments but 3 were given
```

Also, `test_api_function_route` fails because `/predict_image` doesn't work.

@romeran